layers/meta-balena-beaglebone/recipes-core/images/balena-image.inc

```diff
@@ -1,6 +1,11 @@
 IMAGE_FSTYPES:append:beaglebone = " balenaos-img balenaos-img.bmap"
 IMAGE_FSTYPES:append:beaglebone-pocket = " balenaos-img"
 IMAGE_FSTYPES:append:beagleboard-xm = " balenaos-img"
+IMAGE_FSTYPES:append:beagleplay = " balenaos-img"
+IMAGE_FSTYPES:append:beagleplay-k3r5 = " balenaos-img"
```

Comment:
These 2 should not be needed anymore. The fstype is implictily set to `balenaos-img` so you can delete these from here

Reply:
OK.

Reply:
Done.